frontend/src/css/modules/sidebar.scss

```diff
@@ -1,12 +1,13 @@
 .ev-sidebar {
     display: block;
-    height: 100vh;
+    height: calc(100vh - 120px);
```

Comment:
Hey @aka-jain The dubious situation is the subtraction of 120px from 100% of viewport height.
Why we need to subtract this. The height of sidebar should always be equal to the viewport height.
Taking in the case of mobiles where this 120px ( CSS ) will scale up by the `<meta content='width-device-width >' value and then get subtracted from the viewport of Mobiles.
IMHO, 120px creates a dubious situation hence must be removed.